src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -43,8 +43,9 @@ limitations under the License.
 
   <struct name="NOCStruct">
     <cluster code="0x003E"/>
-    <item name="FabricIndex" type="INT8U"/>
+    <item name="FabricIndex" type="fabric_idx"/>
     <item name="NOC" type="OCTET_STRING"/>
+    <item name="ICAC" type="OCTET_STRING"/>    
```

Comment:
Followup: ICAC should be nullable.

Reply:
https://github.com/project-chip/connectedhomeip/issues/13769